config.json

```diff
@@ -1,22 +1,22 @@
 {
-	"version": "1.12.0",
+    "version": "1.13.0",
```

Comment:
IT actually looks like we haven't released 1.12 yet,  so no need to update the version